src/scheduleMessage.js

```diff
@@ -49,10 +74,33 @@ async function startScheduling({ app, usergroups, userCache }) {
                         usergroups.generatePlaintextString(usergroupId),
                         userCache.generatePlaintextString,
                     );
-                    helper.postMessage(app, obj.channel_id, message);
+                    helper.postMessage(app, channelId, message);
                 });
             }
         });
+        // add the job to the map so that we can reschedule it later
+        jobs.set(channelId, job);
+    }
+}
+
+/**
+* Schedules jobs to send a daily message to every channel the bot is a member of.
+*/
+async function startScheduling({ app, usergroups, userCache }) {
+    let channels = await helper.getMemberChannelIds(app);
+    channels = channels.map((channel) => ({
+        channel_id: channel,
+    }));
+    await service.addAllJobs(channels); // add all those channels that are not in the db yet
```

Comment:
Tämä saattaisi ehkä pikavilkaisulla ylikirjoittaa kanavakohtaisen ajan. Pitänee tutkia asiaa.

Reply:
Ahaa, ei näköjään ylikirjoita. "updateOnDuplicate: Fields to update if row key already exists"
https://sequelize.org/v5/class/lib/model.js~Model.html#static-method-bulkCreate